Sources/Sentry/SentrySDK.m

```diff
@@ -39,15 +39,20 @@
 #    import "SentryProfiler+Private.h"
 #endif // SENTRY_TARGET_PROFILING_SUPPORTED
 
+NSString *const SENTRY_XCODE_PREVIEW_ENVIRONMENT_KEY = @"XCODE_RUNNING_FOR_PREVIEWS";
+
 @interface SentrySDK ()
 
 @property (class) SentryHub *currentHub;
 
+#if TEST || TESTCI
+@property (nonatomic, strong, class) NSDictionary<NSString *, NSString *> *processInfoEnvironment;
```

Comment:
can we instead make this part of `TestSentryNSProcessInfoWrapper`? I think if we do that, then we don't have to do the little dance at https://github.com/getsentry/sentry-cocoa/pull/4601/files#diff-1b8a7e8fc89dbafd6be9e731ae59c015f849bb96967cca64c665873b6fdecc1cR209-R219 and https://github.com/getsentry/sentry-cocoa/pull/4601/files#diff-1b8a7e8fc89dbafd6be9e731ae59c015f849bb96967cca64c665873b6fdecc1cR633-R646

Reply:
Yes. It makes sense.